src/main/java/com/untamedears/humbug/Humbug.java

```diff
@@ -1531,6 +1531,73 @@ public void onPlayerBucketEmptyEvent(PlayerBucketEmptyEvent e) {
       }
     }
   }
+  
+  @BahHumbug(opt="drop_ores_as_ore", def="true")
+  @EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
+  public void orebreak(BlockBreakEvent e) {
+	  if(!config_.get("drop_ores_as_ore").getBool()) {
+		  return;
+	  }
+	  if (isOre(e.getBlock().getType())) {
+		  e.setCancelled(true);
```

Comment:
On a second thought setting the block to air without canceling the event will still fuck up logging done in MONITOR priority since you changes the type of the block. I guess we can add the logging here but thats a bit hacky and won't be compatible with other plugins logging ore breaks. No other way to only change the drops from the block?